meta-genivi-dev/recipes-sota/python-pygobject/python-pygobject_2.28.3.bb

```diff
@@ -0,0 +1,55 @@
+# Copied from http://git.yoctoproject.org/cgit/cgit.cgi/poky/tree/meta/recipes-devtools/python?h=jethro d462015
+
+SUMMARY = "Python GObject bindings"
+SECTION = "devel/python"
+LICENSE = "LGPLv2.1"
+
+RECIPE_NO_UPDATE_REASON = "Newer versions of python-pygobject depend on gobject-introspection which doesn't cross-compile"
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=a916467b91076e631dd8edb7424769c7"
+DEPENDS = "python python-pygobject-native libffi glib-2.0"
+DEPENDS_class-native = "python-native libffi-native glib-2.0-native"
+RDEPENDS_class-native = ""
```

Comment:
@chbae Not sure if you saw my earlier comment, but I tried removing this python-pygobject_2.28.3.bb recipe, and so it would use the upgraded one from poky/meta. However, the genivi-swm was unable to import it. When I ran a python prompt and tried `import gobject` I got the same error. Has something else changed with this package between versions? Do you know if I need to configure something differently to use it?